packages/waku/src/lib/middleware/rsc.ts

```diff
@@ -55,7 +53,6 @@ export function rsc<
     ]);
     const viteServer = await viteCreateServer({
       ...(await viteInlineConfig()),
-      root: joinPath(config.rootDir, config.srcDir),
       optimizeDeps: {
```

Comment:
We need this. Let's add it in this PR.
```suggestion
      base: config.basePath,
      optimizeDeps: {
```

Reply:
Maybe we need to this in all vite configs in the code. We can do that in a separate PR.